test/unit/model/execution/DeliveryExecutionListTest.php

```diff
@@ -251,7 +266,70 @@ public function testAdjustDeliveryExecutionsProgressStringWithCustomOption()
         $this->assertSame('finished', $result[0]['state']['progress']);
     }
 
-    private function getExecutionExample()
+    public function testTimerAdjustedTimeNull(): void
+    {
+        $deliveryExecutions[] = $this->deliveryExecution;
+
+        $deliveryHelperService = new DeliveryExecutionList();
+        $deliveryHelperService->setServiceLocator($this->serviceLocatorMock);
+        /* @noinspection PhpUnhandledExceptionInspection */
+        $result = $deliveryHelperService->adjustDeliveryExecutions($deliveryExecutions);
+        $this->assertNull($result[0]['timer']['adjustedTime']);
```

Comment:
Assertion methods are static in phpunit 8. I think it would be better to call it as `static::assertNull()` and others the same.

Reply:
I don't see why do I need to do so, is that the best practice? it was always available with self or static, but everyone prefer to use $this in the objects and static for static... probably it makes code more flexible?

Reply:
`assertNull` (and all other asserts) is now a static method in phpunit. Because of that IDE always highlights it with message that "Static method should not be called dynamically"

Reply:
I guess that this opportunity was there, but we used self:: for the static only, so probably it should be static now, but it doesn't replace $this of phpunit